src/hotspot/share/opto/vtransform.cpp

```diff
@@ -104,7 +104,7 @@ bool VTransformGraph::schedule() {
   }
 
 #ifndef PRODUCT
-  if (_trace._verbose) {
+  if (_trace._info) {
     print_schedule();
```

Comment:
Verbose is often too much, but it is nice to see which `VTransformNode`s are generated, and to see their order after scheduling.